dhctl/pkg/kubernetes/actions/deckhouse/install.go

```diff
@@ -174,12 +175,25 @@ func CreateDeckhouseManifests(kubeCl *client.KubernetesClient, cfg *Config) erro
 			Name:     `ConfigMap "deckhouse"`,
 			Manifest: func() interface{} { return manifests.DeckhouseConfigMap(cfg.DeckhouseConfig) },
 			CreateFunc: func(manifest interface{}) error {
-				_, err := kubeCl.CoreV1().ConfigMaps("d8-system").Create(context.TODO(), manifest.(*apiv1.ConfigMap), metav1.CreateOptions{})
-				return err
+				// check cm existing for prevent error
+				// deckhouse create manifests: create resource: admission webhook "validate-cm.deckhouse-config-webhook.deckhouse.io" denied the request:
+				// changing ConfigMap/deckhouse is not allowed for kubernetes-admin. Use ModuleConfig resources to configure Deckhouse.
+				// after restart bootstrap
+				cm := manifest.(*apiv1.ConfigMap)
+				_, err := kubeCl.CoreV1().ConfigMaps("d8-system").
+					Get(context.TODO(), cm.GetName(), metav1.GetOptions{})
+				if err != nil {
+					if k8serror.IsNotFound(err) {
```

Comment:
You can call Create as before and return nil if error is k8serror.IsAlreadyExists.

See https://github.com/deckhouse/deckhouse/blob/5e37bfea32f51883df0b713ca5a4608267a41c3e/dhctl/pkg/kubernetes/client/state_cache.go#L137

Reply:
good reason, i've fixed